.github/ISSUE_TEMPLATE.md

```diff
@@ -1,10 +1,38 @@
 <!--
 
-Bug report?                   https://RESissues.reddit.com/
-Feature request or question?  https://Enhancement.reddit.com/
-Security issue?               https://www.reddit.com/message/compose?to=/r/Enhancement
+Are you reporting a bug?
+Visit https://RESissues.reddit.com/
+
+
+Are you asking "how do I make RES do something"?
+Visit https://Enhancement.reddit.com/
+
+
+
+Are you requesting a feature for RES?
+Visit https://Enhancement.reddit.com/
+
+
+
+Are you reporting a security issue?
+Visit https://www.reddit.com/message/compose?to=/r/Enhancement
+
+
+
+If you're not sure...
+Visit https://Enhancement.reddit.com/
+
+
```

Comment:
Yes, I'm attempting to put focus on the "post in Reddit" options by pushing the real issue template below the fold. Maybe I should put one line just above the fold with the old message "If you really want to post a GitHub issue..."?